src/hooks/use-brukeraktivitet-lytter.tsx

```diff
@@ -0,0 +1,29 @@
+import {useEffect} from 'react';
+import throttle from 'lodash.throttle';
+
+const TI_SEKUNDER_I_MILLISEKUNDER = 10000;
+
+export const useBrukeraktivitetLytter = (
+    element: Element | Document = document,
```

Comment:
Kunne typen vært `HTMLElement | Document` for å unngå `instanceof` sjekk under?

Reply:
Sjekken på `HTMLElement`/`Element` er forsåvidt overflødig men eg måtte ha det med når eg først sjekket for `Document`. Årsaken til sjekken er egentlig for å håndtere edge-caser der `document` globalen ikkje er tilgjengelig, men dette er antakeligvis ikkje relevant når koden uansett skal køyrast i browser-kontekst. Så eg trur eg berre fjerner heile sjekken.